openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -0,0 +1,196 @@
+# Azure Landing Zones paths
+paths:
+  /api/azure/landingzones/v1:
+    post:
+      summary: Create Azure landing zone
+      operationId: createAzureLandingZone
+      tags: [ LandingZones ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateAzureLandingZoneRequestBody'
+      responses:
+        '200':

```

Comment:
Why have both 200 and 202 responses?

Reply:
In case we immediately get response with landing zone. I saw we have the same pattern for cloud context creation.

Reply:
This is the pattern for async APIs.